src/lib-auth/auth-scram-server.h

```diff
@@ -96,7 +96,7 @@ void auth_scram_server_init(struct auth_scram_server *server_r, pool_t pool,
 void auth_scram_server_deinit(struct auth_scram_server *server);
 
 /* Returns TRUE if authentication was concluded successfully. */
-bool auth_scram_server_acces_granted(struct auth_scram_server *server);
+bool auth_scram_server_access_granted(struct auth_scram_server *server);
```

Comment:
I'm told this function isn't used yet but will be soon, which means now is the right time to fix its name.